.github/workflows/buildswpackage.yaml

```
@@ -0,0 +1,58 @@
+name: Build Shopware Store package
+on:
+  push:
+    branches:
+      - stable      
+      - swpackage-test
+
+jobs:  
+  swpackage:
+    name: Create a new package for shopware
+    runs-on: ubuntu-latest    
+    steps:
+      - name: Setup PHP
+        uses: shivammathur/setup-php@v2
+        with:
+            php-version: 7.4           
+
+      - name: Checkout github repo
+        uses: actions/checkout@v2
+        with:
+          path: MakairaConnect
+
+      - name: Checkout code repo  
+        working-directory: ./MakairaConnect
+        run: git checkout
+
+      - name: Copy .git
+        run: cp -r  ./MakairaConnect/.git .
+          
+      - uses: actions-ecosystem/action-get-latest-tag@v1                
+        id: get-latest-tag
+        
+      - name: Install Dependencies
+        working-directory: ./MakairaConnect
+        run: |
+          sed -i '/shopware\/shopware/d' composer.json >> composer.json
+          composer install -q --no-ansi --no-interaction --no-scripts --no-progress --prefer-dist  
+        
+      - name: Remove build files
+        working-directory: ./MakairaConnect
+        run: |  
+          rm -rf .github/workflows 
+          rm -rf .git
+          rm .gitignore
+          rm .releaserc.json
+          rm composer.json
+          rm package.json
+          rm yarn.lock
+          rm composer.lock
+
+      - name: Zip artifact for deployment        
+        run: zip ${{ steps.get-latest-tag.outputs.tag }}.zip ./MakairaConnect/* -r
+ 
+      - name: 'Upload package'
+        uses: 'actions/upload-artifact@v2'
+        with:
+          name: ${{ steps.get-latest-tag.outputs.tag }}
+          path: ${{ steps.get-latest-tag.outputs.tag }}.zip
```

Files must end with a newline.

```suggestion
          path: ${{ steps.get-latest-tag.outputs.tag }}.zip

```